R/GenerateRefTables.R

```diff
@@ -147,4 +147,146 @@ UpdateMeasureUnitRef <- function() {
   utils::write.csv(WQXunitRef, file = "inst/extdata/WQXunitRef.csv", row.names = FALSE)
 }
 
+#' Generate Unique Harmonization Reference Table
```

Comment:
This came when I did an upstream pul...not sure why it's in this pull request....

Reply:
I moved this to the Transformations.R file recently because it makes more sense to have there, not sure why it is still here. I'll make sure to review & resolve this